RxSwift/Observables/Breakpoint.swift

```diff
@@ -0,0 +1,182 @@
+//
+//  AsMaybe.swift
+//  RxSwift
+//
+//  Created by Anton Nazarov on 09/06/19.
+//  Copyright © 2017 Krunoslav Zaher. All rights reserved.
+//
+
+import Foundation
+
+extension ObservableType {
+    /**
+     Raises a debugger signal when a provided closure needs to stop the process in the debugger.
+     - seealso: [breakpoint operator on Combine](https://developer.apple.com/documentation/combine/publisher/3204688-breakpoint)
+
+     - parameter onNext: Predicate to invoke for each element in the observable sequence.  Return `true` from this closure to raise `SIGTRAP`, or false to continue.
+     - parameter afterNext: Predicate to invoke for each element after the observable has passed an onNext event along to its downstream. Return `true` from this closure to raise `SIGTRAP`, or false to continue.
+     - parameter onError: Predicate to invoke upon errored termination of the observable sequence. Return `true` from this closure to raise `SIGTRAP`, or false to continue.
```

Comment:
You're writing SIGTRAP everywhere but signaling a SIGINT - which is the right one? I think the former?

Reply:
Changed to SIGTRAP, just for consistency with Combine